.changeset/six-mirrors-call.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react": patch
+---
+
+Fix ButtonGroup layout bug
```

Comment:
Do you think this requires a more descriptive change-log? It might affect existing implementations and people may not be sure why those are failing?